packages/client/src/pages/Map/DestinationCard/destinationCard.scss

```diff
@@ -83,8 +89,22 @@
             overflow: hidden;
             text-overflow: ellipsis;
 
-            > * {
-                margin: 0 0 0.5rem;
+            .dest-info-item {
+                margin-bottom: 0.5rem;
+                display: flex;
+
```

Comment:
Can we use $mid-gray for info text? I just feel default pure black is a little too black hahaha.

Reply:
haha yea sure I will update that